tests/admin_inlines/tests.py

```diff
@@ -505,6 +505,32 @@ def test_inline_media_only_inline(self):
         )
         self.assertContains(response, 'my_awesome_inline_scripts.js')
 
+        jquery_index = response.context["media"]._js.index("admin/js/vendor/jquery/jquery.min.js")
+        inlines_index = response.context["media"]._js.index("admin/js/inlines.min.js")
+        custom_index = response.context["media"]._js.index("custom_number.js")
+
+        self.assertTrue(jquery_index < inlines_index)
+        self.assertTrue(jquery_index < custom_index)
```

Comment:
You could use the more specific `assertLess()` which might result in a nicer message upon failure:

https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertLess

Reply:
Thanks. The test is better now because it does not depend on the exact list of admin media, so I can test for an exact match of the JS assets, not just some relative ordering.